tox.ini

```diff
@@ -2,31 +2,44 @@
 envlist = py{27,34,35,36,py}-{brief,prettyquick,full,custom,benchmark}
 toxworkdir={env:TOX_WORK_DIR:.tox}
 
-passenv=
-  HOME
-  LC_ALL
-  COVERAGE_FILE
-
-setenv=
-  PYTHONWARNINGS={env:PYTHONWARNINGS:error::DeprecationWarning,error::FutureWarning}
-
 [testenv]
-deps =
-    -rrequirements/test.txt
-    benchmark: pytest-benchmark==3.0.0
```

Comment:
(`benchmarks` are a thing of the past, I think)

Reply:
They are - well spotted.  Looks like they were removed from the Makefile at the time but not tox.ini